api/src/main/java/io/strimzi/api/kafka/model/common/template/PodTemplate.java

```diff
@@ -51,10 +51,12 @@ public class PodTemplate implements HasMetadataTemplate, UnknownPropertyPreservi
     private List<Toleration> tolerations;
     private List<TopologySpreadConstraint> topologySpreadConstraints;
     private String priorityClassName;
+    private String priorityClassName2;
```

Comment:
typo`?

Reply:
Yes, not sure where that came from. Removed